routes/users.js

```diff
@@ -10,6 +10,7 @@ const router = app.Router();
 
 router.get('/', (req, res, next) => {
   User.find({})
```

Comment:
What if there are 2 millions users? I think in that case you application would crash. Leave it as is for now but later this should be fixed with filtering by first letters.